spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpRequest.java

```diff
@@ -68,6 +66,15 @@
                    .publishOn(Schedulers.fromExecutor(ctx.contextAwareExecutor()));
     }
 
+    private static URI uri(HttpRequest req) {
+        final String scheme = req.scheme();
+        final String authority = req.authority();
+        // Server side Armeria HTTP request always has the scheme and authority.
+        assert scheme != null;
+        assert authority != null;
+        return URI.create(scheme + "://" + authority + req.path());
```

Comment:
How about using [URI constructor](https://docs.oracle.com/javase/8/docs/api/java/net/URI.html#URI-java.lang.String-java.lang.String-java.lang.String-java.lang.String-)?
```java
return new URI(scheme, authority, path, null);
```

Reply:
As we discussed, this will ignore the fragment part, so we should use `URI.create()`. Thanks!